op-supervisor/supervisor/backend/backend.go

```diff
@@ -441,9 +443,52 @@ func (su *SupervisorBackend) checkAccess(acc types.Access, execAt types.Executin
 	if err != nil {
 		return eth.BlockID{}, err
 	}
+
 	return bl.ID(), nil
 }
 
+func (su *SupervisorBackend) asyncVerifyAccessWithRPC(ctx context.Context, acc types.Access, msgBlockFromDB eth.BlockID) {
+	go func() {
```

Comment:
Generally in Go I believe it's better to defer go-routine usage to the caller. I.e. not wrap it here, and instead call `go su.asyncVerifyAccessWithRPC()` (and add a doc comment that it may be safely called on a different go-routine)